osquery/tables/networking/darwin/wifi_utils.mm

```diff
@@ -10,11 +10,45 @@
 #include <CoreFoundation/CoreFoundation.h>
 
 #include <osquery/core/system.h>
+#include <osquery/core/tables.h>
 #include <osquery/tables/networking/darwin/wifi_utils.h>
+#include <osquery/utils/conversions/darwin/cfdata.h>
+#include <osquery/utils/conversions/darwin/cfnumber.h>
+#include <osquery/utils/conversions/darwin/cfstring.h>
 
 namespace osquery {
 namespace tables {
 
+std::string getPropertiesFromDictionary(const CFDictionaryRef& dict,
```

Comment:
This seems fine, we appear to have this code block in a bunch of `osquery/utils/conversions/darwin` which makes me wonder if we should unify it. But, that could not happen in this PR

Reply:
Yes, there is also a bit of IOKit specific parsing. I will revisit when I am doing the refactor and cleanup next